core/controller/src/main/scala/whisk/core/controller/WebActions.scala

```diff
@@ -483,7 +483,24 @@ trait WhiskWebActionsApi extends Directives with ValidateRequestSize with PostAc
               provide(fullyQualifiedActionName(actionName)) { fullActionName =>
                 onComplete(verifyWebAction(fullActionName, onBehalfOf.isDefined)) {
                   case Success((actionOwnerIdentity, action)) =>
-                    if (!action.annotations.getAs[Boolean]("web-custom-options").exists(identity)) {
+                    val requireWebAuthIsBool = (action.annotations.getAs[Boolean]("require-whisk-auth") != None)
```

Comment:
use `.isTruthy` on annotation instead.

Reply:
@rabbah - i think `isTruthy` would return true for a "true" boolean value, non-zero integers and non-empty strings; whereas this check is just determining if the web action is configured with a boolean (true or false) to determine if the existing `require-whisk-auth` behavior should be enforced.

Reply:
Does that mean that the distinction of new vs. old behavior is determined on the type of the annotation? If it's boolean, use the old way, if it's either string or int use the new way?

Could you add a bit to the description describing old vs. new behavior please?

Reply:
@markusthoemmes that is correct.  i've updated the description to make this more clear.